ts/utils/testWrapper.tsx

```diff
@@ -42,3 +45,35 @@ export const renderScreenWithNavigationStoreContext = <S,>(
 
   return render(component, renderOptions);
 };
+
+export const renderTestingComponent = (component: React.ReactElement) => {
```

Comment:
Why can't we use `renderScreenWithNavigationStoreContext` instead of adding a new utility?